doc/CHANGELOG.txt

```diff
@@ -1,4 +1,6 @@
 Changelog:
+May 2020 - Default Due Dates are now reflected in newly created bugs
```

Comment:
This entry can be moved to the list of "Jul 2019 - EmailReporting-0.11.0"
The date will be corrected when 0.11.0 gets released